playbooks/ceph.yml

```diff
@@ -131,9 +131,10 @@
 
 - name: Bootstrap Ceph and apply spec
   tags: cephadm
-  hosts: edpm[0]
+  hosts: "{{ groups[cifmw_ceph_target | default('computes')] | first }}"
```

Comment:
I believe the adoption job is failing here. Its nodeset has an empty `computes` group https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/adoption.yaml#L18, so maybe the whole things needs a `| default({})`? Although I'm not sure why it fails, since it's supposed to be skipped

Reply:
I think that `|default([])` in case we don't have computes is always a good thing and we shouldn't fail because of that.
I'm curious to know why an `adoption` job is failing in the cephadm playbook. I thought that adoption jobs used to deploy `Ceph` in the source cloud through TripleO standalone, hence I'm not explaining why this playbook is executed.

@cescgina do you have any insight on this part?

Reply:
@fmount you're right, it's not needed and it's not executed. Talking with @pablintino  we think that this playbook is just imported, but not run in the adoption job, IIUC